vulnerablecode/settings.py

```diff
@@ -47,6 +47,7 @@
     'vulnerabilities',
     'rest_framework',
     'django_filters',
+    'django_extensions'
```

Comment:
Is this related to the proposal?

Reply:
No, I had to add that to generate the diagram